chainerx/linalg/__init__.pyi

```diff
@@ -1,4 +1,13 @@
+import typing as tp
+
+
 def inv(a: ndarray) -> ndarray: ...
 
 
+def pinv(a: ndarray, rcond: float=...) -> ndarray: ...
+
+
 def solve(a: ndarray, b: ndarray) -> ndarray: ...
+
+
+def svd(a: ndarray, *args: tp.Any) -> tp.Tuple[ndarray, ...]: ...
```

Comment:
Can we not write out all of the arguments? Also, could you fix the return type now that we don't necessarily return a tuple?